### PR TITLE
Use the core-relay setup in local benchmark script

### PR DESCRIPTION
**Pull request by kantp**
This allows to run local benchmarks involving core, relay, and edge nodes.

### CODE REVIEW

File: core/constants.yaml [hunk 1]
```diff
@@ -176,7 +176,7 @@ benchmark: &benchmark
   <<: *testnet
 
   genesisN: 50000
-  k: 6
+  k: 2
```
Comment (avieth) [inline core/constants.yaml]: Is this an important change? Just curious.

Comment (kantp) [inline core/constants.yaml]: No, it's not important.

The reason I made that I want to be more sensitive to bugs that happen at the transition from one epoch to the next.

File: scripts/bench/topology/topology0.yaml [hunk 2]
```diff
@@ -2,18 +2,24 @@ nodes:
...
     addr: 127.0.0.1
     port: 3001
   node2:
     type: core
     region: undefined
-    static-routes: [[node0], [node1]]
+    static-routes: [[node0], [node1], [node4]]
     addr: 127.0.0.1
     port: 3002
+  node4:
```
Comment (avieth) [inline scripts/bench/topology/topology0.yaml]: No `node3`? Was this intentional? Not a big deal in any case.

File: scripts/launch/demo.sh [hunk 3]
```diff
@@ -97,15 +102,13 @@ while [[ $i -lt $panesCnt ]]; do
...
-  pane="${window}.$i"
-
   if [[ $i -lt $n ]]; then
     tmux send-keys "$(node_cmd $i "$stats" "$stake_distr" "$wallet_args" "$system_start" "$config_dir" $exec_name) --no-ntp" C-m
   else
     # Number of transactions to send per-thread: 300
     # Concurrency (number of threads sending transactions); $CONC
     # Delay between sends on each thread: 500 milliseconds
-    tmux send-keys -t ${pane} "sleep 40s && $(bench_cmd $i "$stake_distr" "$system_start" 300 $CONC 500 neighbours)" C-m
+    tmux send-keys "sleep 40s && $(bench_cmd $i "$stake_distr" "$system_start" 3000 $CONC 500 neighbours)" C-m
```
Comment (avieth) [inline scripts/launch/demo.sh]: Should make the number of transactions per thread (3000) come from a script parameter or environment variable.

File: scripts/bench/topology/topology0.yaml (hunk 2 above)
Comment (kantp) [inline scripts/bench/topology/topology0.yaml]: node3 is an edge node that will subscribe to the relay, so it's not mentioned in the topology file.

File: scripts/launch/demo.sh (hunk 3 above)
Comment (kantp) [inline scripts/launch/demo.sh]: I agree, we should do that.

**Pull request merged**